components/fxa-client/src/http_client.rs

```diff
@@ -3,12 +3,22 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 use crate::{config::Config, error::*};
+use browser_id::{derive_key_from_session_token, hawk_request::HawkRequestBuilder};
```

Comment:
Ugh, this reminds me, I have a forgotten PR somewhere that removed big chunks of this module and renamed it to 'session_token' rather than 'browser_id'. Never mind, I'll have to pick up that thread again another time.

Reply:
ref https://github.com/mozilla/application-services/pull/1005 ?

Reply:
Right, that one